libzenohcpp.rb

```diff
@@ -0,0 +1,79 @@
+require "json"
+
+class Libzenohc < Formula
+  release = JSON.parse(File.read("#{__dir__}/release.json"))[File.basename(__FILE__, ".rb")]
+
+  desc "Zenoh-cpp API (geo-distributed pub/sub/query/storage of data)"
+  homepage "https://zenoh.io"
+
+  on_macos do
+    on_intel do
+      url release["x86_64-url"]
+      sha256 release["x86_64-sha256"]
+    end
+    on_arm do
+      url release["aarch64-url"]
+      sha256 release["aarch64-sha256"]
+    end
+  end
+
+  def install
+    lib.install "lib/pkgconfig/zenohcxx.pc"
+    lib.install "lib/cmake/zenohcxx/zenohcxxConfig.cmake"
+    lib.install "lib/cmake/zenohcxx/zenohcxxConfigVersion.cmake"
+    include.install "include/zenoh.hxx"
```

Comment:
is it possible not to list all the include files, since those are being added quite frequently, and instead just install the whole "include" folder ?

Reply:
I looked up some other examples and it seems to be possible but I haven't tested it yet. Pushed the change in 3c25eb